capstone-frontend/src/App.jsx

```diff
@@ -54,7 +54,7 @@ function App() {
             path="/emergencycontact/:userid"
```

Comment:
Can you fetch all the border data in app.jsx, and pass the whole array to `Home`, or pass a single item to `BorderPage` (based on which border it is)

Reply:
That way you don't have to fetch multiple times/in multiple places.

Reply:
Will look into this.